python/lsst/ts/m2/model.py

```diff
@@ -0,0 +1,849 @@
+# This file is part of ts_m2.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import logging
+import copy
+
+import numpy as np
+import pandas as pd
+
+from time import sleep
+
+from lsst.ts.idl.enums import MTM2
+
+__all__ = ["Model"]
+
+
+class Model(object):
+    def __init__(self, log=None, telemetry_interval=0.05):
+        """Initialize the Model class.
+
+        Parameters
+        ----------
+        log : `logging.Logger` or None, optional
+            A logger. If None, a logger will be instantiated. (the default is
+            None)
+        telemetry_interval : `float`, optional
+            Telemetry interval in second. (the default is 0.05, which means
+            20 Hz)
+
+        Attributes
+        ----------
+        log : `logging.Logger`
+            A logger.
+        telemetry_interval : `float`
+            Telemetry interval in second.
+        inclination_source : `lsst.ts.idl.enums.MTM2.InclinationTelemetrySource`
```

Comment:
You may want to use `MTM2.InclinationTelemetrySource` instead. Since MTM2 is imported here, I think it is ok to omit the `lsst.ts.idl.enums` part.

Reply:
Done in the commit: f20e3d7. Thanks!